CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
It should not do so in the new (LCG_87) setup. So please make an issue so we can follow up after merging, if we merge this one first.